openslides/core/projector.py

```diff
@@ -139,3 +139,10 @@ class Message(ProjectorElement):
     def check_data(self):
         if self.config_entry.get('message') is None:
             raise ProjectorException('No message given.')
+
+
+class ListOfSpeakersOverlay(ProjectorElement):
+    """
+    List of speakers (last, current, next) on the projector. Rendered as overlay.
+    """
+    name = 'core/speakeroverlay'
```

Comment:
See https://github.com/OpenSlides/OpenSlides/issues/2302. For short you may just release all users. We later can change the restrictions again.